cypher/cypher-docs/src/docs/dev/ql/administration/security/database/admin-role-database.asciidoc

```diff
@@ -1,78 +0,0 @@
-The administrators can use the following Cypher commands to manage Neo4j database administrative rights.
```

Comment:
neo4j-manual-modeling/4.3/cypherManual/asciidoc/access-control/database-administration.adoc

https://neo4j.com/docs/cypher-manual/4.3/access-control/database-administration/

The introduction.